server/routerlicious/packages/services/src/documentManager.ts

```diff
@@ -44,4 +107,30 @@ export class DocumentManager implements IDocumentManager {
 		);
 		return restWrapper;
 	}
+
+	/**
+	 * Creates a cache key to retreive static data from a document
+	 *
+	 * @param documentId - ID of the document to create an access key for
+	 * @returns - A cache key to access static data for [documentId]
+	 */
+	private static getDocumentStaticKey(documentId: string): string {
+		return `staticData:${documentId}`;
+	}
+
+	/**
+	 * Extracts the static properties from an IDocument
+	 *
+	 * @param document - Document to get properties from
+	 * @returns - The static properties of [document]
+	 */
+	private static getStaticPropsFromDoc(document: IDocument): IDocumentStaticProperties {
+		return {
```

Comment:
Maybe we can use the spread operator here?

`{ ...document }`

Reply:
Resolved in call